src/less/tooltip/ds6/tooltip.less

```diff
@@ -0,0 +1,404 @@
+
+        &::after {
+            border-style: solid;
+            border-width: 8px;
+            content: "";
+            position: absolute;
+        }
+    }
+
+    &__body {
+        margin: 16px 16px 16px 16px;
+    }
+}
+
+.tooltip {
+    &:hover {
+        .tooltip__top-arrow,
+        .tooltip__top-leftarrow,
+        .tooltip__top-rightarrow,
+        .tooltip__bottom-arrow,
+        .tooltip__bottom-leftarrow,
+        .tooltip__bottom-rightarrow,
+        .tooltip__left-arrow,
+        .tooltip__left-toparrow,
+        .tooltip__left-bottomarrow,
+        .tooltip__right-arrow,
+        .tooltip__right-toparrow,
+        .tooltip__right-bottomarrow {
+            visibility: visible;
+        }
+    }
+
+    &__bottom-arrow,
+    &__bottom-leftarrow,
+    &__bottom-rightarrow,
+    &__top-arrow,
+    &__top-leftarrow,
+    &__top-rightarrow,
+    &__left-arrow,
+    &__left-toparrow,
+    &__left-bottomarrow,
+    &__right-arrow,
+    &__right-toparrow,
+    &__right-bottomarrow {
+        width: 344px;
+
+        &::after {
+            border-color: @ds6-color-p002-blue;
+            box-shadow: 0 0 0 @ds6-color-p002-blue;
+            box-sizing: border-box;
+        }
+    }
+
+    &__bottom-arrow,
+    &__bottom-leftarrow,
+    &__bottom-rightarrow {
+        bottom: 100%;
+        margin-bottom: 16px;
+
+        &::after {
+            top: 100%;
+        }
+    }
+
+    &__top-arrow,
+    &__top-leftarrow,
+    &__top-rightarrow {
+        margin-top: 16px;
+
+        &::after {
+            top: 2%;
+        }
+    }
+
+    &__left-arrow,
+    &__left-toparrow,
+    &__left-bottomarrow {
+        left: 110%;
+        margin-left: 16px;
+
+        &::after {
+            margin-left: -10px;
+        }
+    }
+
+    &__right-arrow,
+    &__right-toparrow,
+    &__right-bottomarrow {
+        margin-right: 16px;
+        right: 100%;
+
+        &::after {
+            left: 95%;
+            margin-left: 5px;
+        }
+    }
+
+    &__bottom-arrow,
+    &__top-arrow {
+        margin-left: -161px;
+
+        &::after {
+            left: 45%;
+            margin-left: 5px;
+        }
+    }
+
+    &__bottom-leftarrow,
+    &__top-leftarrow {
+        margin-left: -24px;
+
+        &::after {
+            left: 7%;
+        }
+    }
+
+    &__bottom-rightarrow,
+    &__top-rightarrow {
+        margin-left: -300px;
+
+        &::after {
+            left: 87%;
+        }
+    }
+
+    &__left-arrow,
+    &__right-arrow {
+        top: -100%;
+
+        &::after {
+            top: 50%;
+        }
+    }
+
+    &__left-toparrow,
+    &__right-toparrow {
+        top: -50%;
+
+        &::after {
+            top: 35%;
+        }
+    }
+
+    &__left-bottomarrow,
+    &__right-bottomarrow {
+        top: -140%;
+
+        &::after {
+            top: 65%;
+        }
+    }
+
+    &__window {
+        background-color: @ds6-color-p002-blue;
+        color: @ds6-color-g201-gray;
+        visibility: hidden;
+        width: 344px;
```

Comment:
We also need the 400px version for larger screens.  Use media-queries.  And can we use max-width instead of width?